data/hazards.txt

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2020 by Zitchas
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+hazard "Ember Waste Base Heat"
+	"duration" 1 200
+	"strength" 0.25 2
+	"range" 20000
+	"environmental effect" "heat hazard" 200
+	weapon
+		"relative heat damage" 0.00035
+
+
+
+effect "heat hazard"
+	sprite "effect/ew heat/ew heat"
```

Comment:
You shouldn't need a nested folder for these sprites.

Reply:
But it makes things so much tidier... (I'd honestly like to  put *all* the effects in their own folders, but I figure some people would object because it just looks like churn. No reason to not start being tidy now, though. I'd be happy to sort them all into folders, though.)